src/components/form/range/dual_range.tsx

```diff
@@ -418,6 +418,11 @@ export class EuiDualRange extends Component<EuiDualRangeProps> {
 
     const minInput = !!showInput ? (
       <EuiRangeInput
+        // Overridable props
+        aria-describedby={this.props['aria-describedby']}
+        aria-label={this.props['aria-label']}
+        {...minInputProps}
```

Comment:
This is the order that I was trying to mention before. All the props below this line are mandator and therefore cannot be allowed to be overridden by `...inputProps`. The only way to ensure this is that `...inputProps` comes **before** the mandator props.